.travis.yml

```diff
@@ -8,17 +8,20 @@ android:
     # - platform-tools
 
     # The BuildTools version used by your project
-    - build-tools-28.0.3
+    - build-tools-29.0.1
 
     # The SDK version used to compile your project
-    - android-28
+    - android-29
 
     # Additional components
     - extra-google-google_play_services
     - extra-google-m2repository
     - extra-android-m2repository
-
     # Specify at least one system image,
     # if you need to run emulator(s) during your tests
     - sys-img-x86-android-28
-#    - sys-img-armeabi-v7a-android-17
\ No newline at end of file
+#    - sys-img-armeabi-v7a-android-17
+
+ licenses:
```

Comment:
What are these licenses for?

Reply:
I was following the documentation of travis I am not yet sure if it is needed or not.